### PR TITLE
openvino: added telemetry support

### PR DESCRIPTION
**Pull request by ilya-lavrenov**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by ilya-lavrenov** [comment]
> ` Errno::ENOTEMPTY: Directory not empty @ dir_s_rmdir - /private/tmp/homebrew-unpack20240408-6500-fit0gj`

I don't understand what does it mean. Could you please advice what I'm doing wrong?
I have not found good example in existing brew formulas, where C++ extension is built and other python packages (like `packaging`) are used

**Comment by chenrui333** [comment]
> > ` Errno::ENOTEMPTY: Directory not empty @ dir_s_rmdir - /private/tmp/homebrew-unpack20240408-6500-fit0gj`
> 
> I don't understand what does it mean. Could you please advice what I'm doing wrong? I have not found good example in existing brew formulas, where C++ extension is built and other python packages (like `packaging`) are used

yeah, not quite sure what happened, gonna rebase and kick a new run.

File: Formula/o/openvino.rb [hunk 1]
```diff
@@ -75,6 +76,16 @@ class Openvino < Formula
...
 
+  resource "openvino-telemetry" do
+    url "https://files.pythonhosted.org/packages/37/dd/675a6349e4b5a1d5bfaab940cd52a70c988099b6f4c1689af1884c49815a/openvino-telemetry-2023.2.1.tar.gz"
+    sha256 "ca2106b84671c6edfdc562c03b1c6dd3434d649a4c2174aa47f4628aa66e660a"
+  end
+
+  resource "packaging" do
+    url "https://files.pythonhosted.org/packages/ee/b5/b43a27ac7472e1818c4bafd44430e69605baefe1f34440593e0332ec8b4d/packaging-24.0.tar.gz"
+    sha256 "eb82c5e3e56209074766e6885bb04b8c38a0c015d0a30036ebe7ece34c9989e9"
+  end
```
Comment (ilya-lavrenov) [inline Formula/o/openvino.rb:87]: can someone help with embedding these packages as part of OpenVINO Python API?

**Comment by ilya-lavrenov** [comment]
I need help here, because I don't know how to add extra python packages to OpenVINO Python API

File: Formula/o/openvino.rb (hunk 1 above)
Comment (ilya-lavrenov) [reply Formula/o/openvino.rb:86]: basically, I don't know how to add `python-*` packages here
We have some OV python specific content and need to add several standard modules on top of it.

**Comment by ilya-lavrenov** [comment]
Looks like OpenVINO has been built, thank you @p-linnane and @branchvincent 
Do we need extra labels to pass OpenCV tests? long-run or something similar?

**Comment by branchvincent** [comment]
Needs #175545 first, then we should be good

**Review by jaraco — COMMENTED** [review]
LGTM

**Review by chenrui333 — APPROVED** [review]

**Comment by branchvincent** [comment]
thanks so much @ilya-lavrenov! sorry this took so long, and thank you @jaraco for your bug report and review here

**Pull request merged**